src/components/PaymentCard/PaymentCardStyled.js

```diff
@@ -98,46 +72,56 @@ export const CardExpirationDateStyled = styled.div.attrs(() => ({
 export const CardEditStyled = styled.button.attrs(() => ({
   className: 'msd__payment-method__button'
 }))`
-  position: absolute;
-  bottom: 18px;
-  right: 16px;
   color: ${White};
-  z-index: 2;
 
-  padding: 9px 14px;
-  background-color: ${ConfirmColor};
-  font-size: 11px;
+  padding: 11px 25px;
+  background-color: ${FontColor};
+  font-size: 14px;
+  line-height: 16px;
   font-weight: 600;
-  border-radius: 17px;
+  border-radius: 21px;
   border: 0;
-  box-shadow: 0px 3px 50px #00000014;
 
   &:hover {
     cursor: pointer;
   }
 `;
 
-export const CardWrapStyled = styled.div.attrs(() => ({
-  className: 'msd__payment-card__wrapper'
-}))`
-  width: 265px;
-  font-family: Arial, Helvetica, sans-serif;
-
-  ${props =>
-    props.type !== '' &&
-    css`
-      ${CardStyled} {
-        background-color: ${paymentMethodColors[`${props.type}FontColor`]};
-
-        &:after {
-          background-color: ${paymentMethodColors[
-            `${props.type}SecondaryColor`
-          ]};
-        }
-      }
-    `}
+export const CardInfoStyled = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+
+  ${mediaFrom.smallest`
+    margin-bottom: 0;
+  `}
 `;
 
-export const MethodNameStyled = styled.strong`
-  text-transform: capitalize;
+export const CardDetailsStyled = styled.div`
+  margin-left: 20px;
+  margin-right: auto;
+`;
+
+export const CardDetailsNameStyled = styled.div`
+  color: ${FontColor};
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 20px;
+  margin-right: 4px;
+`;
+
+export const CardDetailsNameWrapStyled = styled.div`
+  display: flex;
+  margin-bottom: 4px;
+`;
+
+export const CardInfoWrapStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
do we need a custom class names for these new elements? I think it may be worth to add